package.json

```diff
@@ -24,6 +24,9 @@
     "build/types/**/*",
     "config/accept.json"
   ],
+  "bin": {
+    "kubernetes-entity-provider": "./bin/kubernetes-entity-provider"
```

Comment:
This could be maybe better in a format like `agent-entity-provider --variant kubernetes` or something